spec/features/admin/merchant_items/create_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
```

Comment:
All admin crud tests were either missing or improperly labeled so had to add all crud tests